### PR TITLE
BZ#1871145 - An IPAM static address must be specified with CIDR

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1871145

Reality and the documentation must match.

### CODE REVIEW

**Comment by jboxman** [comment]
@dougbtv, this is as detailed as I got for this PR; I want to fix the docs bug where the user gets an error.

Does the updated text make sense?

Thanks!

File: modules/nw-multus-ipam-object.adoc [hunk 1]
```diff
@@ -276,13 +276,13 @@ ipamConfig:
 <1> A collection of mappings that define IP addresses to assign to the virtual
 interface. Both IPv4 and IPv6 IP addresses are supported.
 
-<2> An IP address that you specify.
+<2> An IP address and network prefix that you specify. For example, if you specify `10.10.21.10/24`, then the additional network is assigned an IP address of `10.10.21.10` and the netmask is `255.255.255.0`.
```
Comment (dougbtv) [inline modules/nw-multus-ipam-object.adoc:279]: Yep, I think this captures the heart of the problem, that we're kind of using CIDR format to express two things here. And also really like that you explicitly call out that you're getting `10.10.21.10` as your IP address :+1:

**Review by dougbtv — APPROVED** [review]

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**